cpp_guidelines/.clang-format

```diff
@@ -0,0 +1,107 @@
+---
+Language: Cpp
+Standard: c++17
+DisableFormat:   false
+AccessModifierOffset: -4
+AlignAfterOpenBracket: Align
+AlignConsecutiveAssignments: true
+AlignConsecutiveBitFields: true
+AlignConsecutiveDeclarations: false
+AlignConsecutiveMacros: true
+AlignEscapedNewlines: Right
+AlignOperands: true
+AlignTrailingComments: true
```

Comment:
**vote**
|-current| false |
|--|--|
|0|1|

Vote for changing to `false` to avoid strange alignments for line ending comment in two or more lines e.g. namespace comments.

# Example

current/true):
```C++
    namespace dim
    {
        namespace traits
        {
            //#############################################################################
            //! The CPU OpenMP 2.0 block accelerator dimension getter trait
            //! specialization.
            template <typename TDim, typename TIdx>
            struct DimType<acc::AccCpuOmp2Blocks<TDim, TIdx>>
            {
                using type = TDim;
            };
        } // namespace traits
    }     // namespace dim
```
suggestion:
```C++
    namespace dim
    {
        namespace traits
        {
            //#############################################################################
            //! The CPU OpenMP 2.0 block accelerator dimension getter trait
            //! specialization.
            template <typename TDim, typename TIdx>
            struct DimType<acc::AccCpuOmp2Blocks<TDim, TIdx>>
            {
                using type = TDim;
            };
        } // namespace traits
    } // namespace dim
```

Reply:
Well, if we had no namespace comments, this problem would not exist :D

Reply:
current | false
-- | --
0 | 2

Reply:
current | false
-- | --
0 | 2

Reply:
current | false |
--------|-----------|
 0      |   3

Reply:
current | false
-- | --
0 | 5

Reply:
current | false |
--------|-----------|
 0      |   6

This looks like a bug in clang-format to me. Nevertheless, we should remove this as long as this behaviour exists.

Reply:
current | false
-- | --
0 | 7

Reply:
current | false
-- | --
0 | 8

Reply:
Reminder: Voting will be closed on 2020-09-26 at 13:38 HZDR time.

Reply:
Voting closed, the winner is `false`.